### PR TITLE
Switch back to formal versions of `invoke` and `fabric`

### PR DESCRIPTION
**Pull request by fruch**
since those two PR were merged by Jeff:
* https://github.com/pyinvoke/invoke/pull/645
* https://github.com/fabric/fabric/pull/1989

### CODE REVIEW

File: docker/env/version [hunk 1]
```diff
@@ -1 +1 @@
-0.40
+0.43
```
Comment (bentsi) [inline docker/env/version]: why did you move 3 versions up?

Comment (fruch) [reply docker/env/version]: cause seem like we use those number before in some PR, and i'm failing in CI cause of that. (i.e. cause of images already there)
